### PR TITLE
Various fixes/improvements

### PR DESCRIPTION
**Pull request by jsidorenko**
1) Added support for attributes. You simply pass the array of attributes you want to attach to each NFT, and the script will take them from CSV file, ignoring the empty values.
2) added support for `startItemId` param to start minting from the e.g. ID=1
3) fixed a bunch of typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +69,7 @@ Note:
 - If the calculated row numbers fall outside of the number of rows in the csv file (e.x. `offset+count-1 > last_row_number_in_the_file`) the minting will stop after the last row number.
 - The `item.metadata.imageFile` specifies the file path that contains the media file that is going to be minted.
 - The `item.metadata.videoFile` specifies the file path that contains the video file that is going to be minted. Has the same naming format as `item.metadata.imageFile`.
-- the values surrounded by `<<` and `>>` will be filled from the collumns of the data `.csv` file. e.g. fo the
+- the values surrounded by `<<` and `>>` will be filled from the columns of the data `.csv` file. e.g. fo the
```
Comment (VadimSaveljev) [inline README.md]: fo => for

File: src/workflow/wfConfig.js [hunk 2]
```diff
@@ -83,7 +83,13 @@ const parseConfig = (cfile) => {
     if (!isEmptyObject(itemMetadata)) {
       if (itemMetadata.imageFile) {
         const parts = itemMetadata.imageFile.split('/');
-        const imageFileNameTemplate = parts.pop();
+        let imageFileNameTemplate = parts.pop();
+
+        // support the template data in the last folder's name, e.g. "some_path/<<Column1>>_<<Column2>>/<<File name>>";
+        const [last] = parts.slice(-1);
+        if (last.includes('<<') &&last.includes('>>')) {
```
Comment (VadimSaveljev) [inline src/workflow/wfConfig.js]: && last

**Review by VadimSaveljev — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jsidorenko) [inline README.md]:
```suggestion
- the values surrounded by `<<` and `>>` will be filled from the columns of the data `.csv` file. e.g. for the
```

File: src/workflow/wfConfig.js (hunk 2 above)
Comment (jsidorenko) [inline src/workflow/wfConfig.js]:
```suggestion
        if (last.includes('<<') && last.includes('>>')) {
```

**Pull request merged**